pkgs/applications/window-managers/sway/default.nix

```
@@ -1,23 +1,22 @@
 { lib, stdenv, fetchFromGitHub, fetchpatch, substituteAll, swaybg
 , meson, ninja, pkg-config, wayland-scanner, scdoc
 , wayland, libxkbcommon, pcre2, json_c, libevdev
-, pango, cairo, libinput, libcap, pam, gdk-pixbuf, librsvg
+
+, pango, cairo, libinput, gdk-pixbuf, librsvg
```

```suggestion
, pango, cairo, libinput, gdk-pixbuf, librsvg
```

nit: That random empty line seems weird - hope it's fine if I use GH with squash and merge to remove it.

---

That would break the signature on the commit.